src/c_kzg_4844.c

```diff
@@ -1253,66 +1258,86 @@ C_KZG_RET compute_blob_kzg_proof(
 }
 
 /**
- * Given a blob and its proof, verify that it corresponds to the provided
- * commitment.
- *
- * @param[out] ok               True if the proofs are valid, otherwise false
+ * Given a blob and its proof and commitment as bytes, compute a
+ * kzg_evaluation_claim by parsing these into group elements and computing the
+ * Fiat-Shamir challenge for verification. The resulting claim consists of a
+ * commitment to a polynomial p, an evaluation point, a puported evaluation
```

Comment:
I think it's better not to repeat what a claim consists of here because if we add a new field into the claim struct, we need to update the doc everywhere.